test/unit/remote-config/remote-config.spec.ts

```diff
@@ -596,6 +614,18 @@ describe('RemoteConfig', () => {
           }).to.throw(
             'Cannot set property etag of #<RemoteConfigTemplateImpl> which has only a getter');
 
+          const v = template.version;
+          expect(v).to.be.not.undefined;
+          const version = v as Version;
+          expect(version.versionNumber).to.equal('86');
```

Comment:
Can we do this:

```
expect(version).to.deep.equal(VERSION_INFO);
```

Reply:
`version` is a `Version` type and `VERSION_INFO` is a js object so deep equal didn't work unfortunately...

Reply:
An alternative is to define a global `Version` const an use that here

Reply:
If that's feasible let's do that. Otherwise the way it is now is not terrible either.

Reply:
Looks like it wouldn't work unless we use an instance of the internal `VersionImpl`, which I do not think we can use here.